pages/common/terraform-plan.md

```diff
@@ -0,0 +1,32 @@
+# terraform plan
+
+> Generate and show Terraform execution plans.
+> More information: <https://www.terraform.io/docs/cli/commands/plan.html>.
+
+- Generate and show the execution plan in the currently directory;
+
+`terraform plan`
+
+- Show a plan to destroy all remote objects that currently exist:
+
+`terraform plan -destroy`
+
+- Show a plan to update the Terraform state and output values:
+
+`terraform plan -refresh-only`
+
+- Specify values for input variables:
+
+`terraform plan -var '{{name1=value1}}' -var '{{name2=value2}}'`
```

Comment:
```suggestion
`terraform plan -var '{{name1}}={{value1}}' -var '{{name2}}={{value2}}'`
```
In this case we don't put the `=` in token syntax, because it is nothing the user can change

Reply:
Thanks @marchersimon 
I'll fix it and wait for another approval👍